transform/transformer_test.go

```diff
@@ -104,24 +105,65 @@ func TestLogTransformer_AddHeader(t *testing.T) {
 			transformer: LogTransformer{
 				LogContent: "+[+] helloworld",
 				TagID:      "some github diff",
-				Vcs: "github",
+				Vcs:        "github",
 			},
 			expected: "\n## cdk diff for some github diff \n<details>\n<summary>Click to expand</summary>\n\n```diff\n+[+] helloworld\n```\n</details>\n",
 		},
 		// when using vcs github but setting disable-collapse it should have no collapsible section
 		{
 			transformer: LogTransformer{
-				LogContent: "+[+] helloworld",
-				TagID:      "some github diff",
-				Vcs: "github",
+				LogContent:      "+[+] helloworld",
+				TagID:           "some github diff",
+				Vcs:             "github",
 				DisableCollapse: true,
 			},
 			expected: "\n## cdk diff for some github diff \n\n```diff\n+[+] helloworld\n```\n",
 		},
+		// when displaying overview for number of differences
+		{
+			transformer: LogTransformer{
+				LogContent:      "Number of stacks with differences: 2",
+				TagID:           "some github diff",
+				Vcs:             "github",
+				DisableCollapse: true,
+				ShowOverview:    true,
+			},
+			expected: "\n## cdk diff for some github diff \n\n\n```diff\nNumber of stacks with differences: 2\n```\n",
+		},
+		// when displaying overview for number of replaces
+		// {
+		// 	transformer: LogTransformer{
+		// 		LogContent:      "[~] AWS::DynamoDB::Table ddb-table ddbtable7F3F6F3F replace\n └─ [~] TableName (requires replacement)\n-    ├─ [-] ddb-second-table\n+    └─ [+] ddb-second-table2",
+		// 		TagID:           "some github diff",
+		// 		Vcs:             "github",
+		// 		DisableCollapse: true,
+		// 		ShowOverview:    true,
+		// 	},
+		// 	// contains: "⚠️  Number of resources that require replacement: 2",
+		// 	contains: "Number",
+		// },
+		{
+			transformer: LogTransformer{
+				LogContent:      "[~] AWS::DynamoDB::Table ddb-table ddbtable7F3F6F3F replace\n └─ [~] TableName (requires replacement)\n-    ├─ [-] ddb-second-table\n+    └─ [+] ddb-second-table2ein\n⚠️zwei\ndrei",
+				TagID:           "some github diff",
+				Vcs:             "github",
+				DisableCollapse: true,
+				ShowOverview:    true,
+			},
+			// contains: "⚠️  Number of resources that require replacement: 2",
+			contains: "⚠️zwei",
+		},
 	}
 	for _, c := range cases {
 		c.transformer.addHeader()
-		assert.Equal(t, c.expected, c.transformer.LogContent)
+		// check if expected or contains is set
+		if c.expected != "" {
+			assert.Equal(t, c.expected, c.transformer.LogContent)
+		}
```

Comment:
let's revert all changes on this test function and move it out to `TestLogTransformer_TransformDiffAddHeader`

Reply:
it's removed